MoreLinq.Test/OrderedMergeTest.cs

```diff
@@ -0,0 +1,294 @@
+namespace MoreLinq.Test
+{
+    using System;
```

Comment:
I'm sure resharper used to point this out - my version seems to have stopped doing this.
I'll get it removed.

Reply:
No problem. You don't even need it from ReSharper as [elimination of unused imports it has been part of Visual Studio proper since some time](https://blogs.msdn.microsoft.com/zainnab/2010/06/29/organize-usings-c-only/).